examples/plot_SpecDatabase.py

```diff
@@ -37,7 +37,7 @@
 s1 = sf.eq_spectrum(Tgas=300, path_length=1)
 
 # Creating SpecDatabase
-my_folder = os.getcwd() + "/SpecDatabase_Test"
+my_folder = Path.cwd() / "SpecDatabase_Test"
```

Comment:
I have updated it in my PR using `os.path.dirname(os.path.realpath(__file__)) + "/SpecDatabase_Test"` and also I'm deleting the SpecDatabase folder using `shutil` so that no issue is created locally. So you can undo the changes here to avoid any conflict or maybe the other way around?